### PR TITLE
CRYP-194:  Created list of all contacts for a user 

### PR DESCRIPTION
**Pull request by alexbara2000**
- used a flatlets to create the list of all users
- Added sticky headers to the contact list
- Big thanks to @jason-gerard and @Andre-Ibrahim for all the help

image

### CODE REVIEW

File: packages/client/screens/ContactsListScreen.tsx [hunk 1]
```diff
@@ -1,5 +1,69 @@
...
+import { useIsFocused } from "@react-navigation/native";
 
 export default function ContactsListScreen() {
-    return <></>;
+    const contactsGateway = new ContactsGateway();
+
+    const isFocused = useIsFocused();
+
+    const { token, user } = React.useContext(AuthContext);
+    const [contactsWithHeader, setContactsWithHeader] = React.useState<any[]>([]);
```
Comment (Andre-Ibrahim) [inline packages/client/screens/ContactsListScreen.tsx:15]: any? Javascript? I would make a type thats has contacts with the headers field

**Review by Andre-Ibrahim — APPROVED** [review]
Looks good

**Review by domsec — APPROVED** [review]
Great work

**Review by jason-gerard — APPROVED** [review]
Very cool

**Review by narroarrow — APPROVED** [review]

**Pull request merged**